config/metadata/configMeta.yaml

```diff
@@ -1949,19 +1949,6 @@ groups:
           happen is if a Refinery crashes or is forcibly shut down. TODO: not
           sure we need to expose this.
 
-      - name: MaxTraceRetention
```

Comment:
Lets update TraceTime's configuration to state that it is also the value used for Redis retention